### PR TITLE
Add Dep app

### PR DESCRIPTION
**Pull request by ahmed-taj**
Adds https://github.com/ahmed-taj/dep to apps list

### CODE REVIEW

File: _apps/dep.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```
Comment (bkeepers) [inline _apps/dep.md]: The rendering on this section doesn't quite look right. I don't have time to dig into it right now, but would you be willing to see if you can tweak this so it renders properly on the website?

dep___probot

Comment (ahmed-taj) [reply _apps/dep.md]: Will give it a try

**Comment by ahmed-taj** [comment]
@bkeepers I've converted the Markdown table to pure HTML. Not sure that actually solves the problem. 

Please let me know if it doesn't work yet.

**Comment by bkeepers** [comment]
Looks great, thanks!

**Pull request merged**